pkgs/tools/misc/flashrom-stable/default.nix

```diff
@@ -0,0 +1,36 @@
+{ fetchgit
+, installShellFiles
+, lib
+, libftdi1
+, libgpiod
+, libjaylink
+, libusb1
+, pciutils
+, pkg-config
+, stdenv
+}:
+
+stdenv.mkDerivation rec {
+  pname = "flashrom-stable";
+  version = "1.1";
+
+  src = fetchgit {
+    url = "https://review.coreboot.org/flashrom-stable";
+    rev = "272aae888ce5abf5e999d750ee4577407db32246";
+    sha256 = "sha256-DR4PAING69+TMsyycGxt1cu0ba1tTlG36+H/pJ0oP4E=";
+  };
+
+  nativeBuildInputs = [ pkg-config installShellFiles ];
+  buildInputs = [ libftdi1 libgpiod libjaylink libusb1 pciutils ];
+
+  makeFlags = [ "PREFIX=$(out)" "libinstall" ];
+
+  meta = with lib; {
+    homepage = "https://www.flashrom.org";
+    description = "Utility for reading, writing, erasing and verifying flash ROM chips.";
+    license = with licenses; [ gpl2 gpl2Plus ];
```

Comment:
gpl2Plus -> gpl2

```suggestion
    license = with licenses; [ gpl2Plus ];
```

Reply:
I added both by intention. Not everything is licensed with GPL+2 but parts of it.